File: c programs/Assignments/Assignment_2/Assign2_4.c
// Accept two numbers from user and display first number in second number of times.

//Input 12 5
//output 12 12 12 12 12 

//Input -2 3
//Output -2 -2 -2

//Input  21 -3
//Output 21 21 21

//Input -2 0
//output:

#include<stdio.h>

void Display(int iNo,int iFrequency)
{
	int iCnt = 0;
	
	 // Input updator
    if(iFrequency < 0)             // If the input is negative
    {
        iFrequency = -iFrequency;         // convert that negative number into positive number
    }
    
	for(iCnt = 0;iCnt < iFrequency;iCnt++)
	{
		printf("%d \t",iNo);
	}
}
int main()
{
	int iValue = 0;
	int iCount = 0;
	
	printf("Enter number:\n");
	scanf("%d",&iValue);
	
	printf("Enter Frequency:\n");
	scanf("%d",&iCount);
	
	Display(iValue,iCount);
	
	return 0;
}

File: Python/Assignment_2/Assign2_4.py
#4.Write a program which accept one number form user and return addition of its factors. 
#Input : 12 Output : 16 (1+2+3+4+6)

def Factorial_Add(iNO):
          icnt=1
          sum=0
          while iNO>icnt:
                    if iNO%icnt==0:
                              sum=sum+icnt
                    icnt=icnt+1
          return sum

def main():

    iValue=(int(input("Enter the  number")))
	
    print("Sum is",+Factorial_Add(iValue))

if __name__=="__main__":
   main()